model.py

```diff
@@ -607,8 +634,79 @@ def test(self, *args, **kwargs):
     def save(self, *args, **kwargs):
         return self._adapter.save(*args, **kwargs)
 
-    def load(self, *args, **kwargs):
-        return self._adapter.load(*args, **kwargs)
+    def load(self, path, reset_optimizer=False, layers=None, weights=None):
```

Comment:
As discussed on Hi,  change `layers, weights` to a bool

Reply:
Done. Thanks! Use `skip_mismatch` replace `layers, weights`, which can also implement fine-tuning or transfer-learning by skipping the mismatch parameters, and it is simpler.